pkg/nodebootstrap/assets/bootstrap.al2.sh

```diff
@@ -16,6 +16,16 @@ function get_max_pods() {
 NODE_IP="$(curl --silent http://169.254.169.254/latest/meta-data/local-ipv4)"
 INSTANCE_ID="$(curl --silent http://169.254.169.254/latest/meta-data/instance-id)"
 INSTANCE_TYPE="$(curl --silent http://169.254.169.254/latest/meta-data/instance-type)"
+MACHINE=$(uname -m)
+
+if [ "$MACHINE" == "x86_64" ]; then
+    ARCH="amd64"
+elif [ "$MACHINE" == "aarch64" ]; then
+    ARCH="arm64"
+else
+    echo "Unknown machine architecture '$MACHINE'" >&2
+    exit 1
```

Comment:
I think It might be better here to use `x86_64` as the default MACHINE here instead of failing the bootstrapping. (With the logging of what is going on of course).

Reply:
thanks @martina-if 
I have updated the script with your suggestion.

Cheers